samples/basic.html

```diff
@@ -1,11 +1,13 @@
 <!doctype html>
-<html>
+<html lang="en">
 
 <head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Basic treemap sample</title>
-	<script src="utils.js"></script>
-	<link rel="stylesheet" type="text/css" href="style.css">
-	</link>
+	<script src="/utils.js"></script>
```

Comment:
do these links actually work with '/' prefix?

Reply:
Absolutely! If `src="utils.js"` is working then it will work with the prefix `/` too. Here this `/` will work as a more strict relative file path for files in the same directory. Also, it's a Google best practice to include `/` while including other resources in the document that lies in the same directory.

Reply:
Unless deployed in a sub folder. But I've been converting the samples to the docs already, so that will not happen.